http://www.zeroc.com/forums/bug-reports/5872-3-5b-cpp-test-icegrid-admin-test-failure.html

bash-4.2$ ./run.py
starting icegrid registry... ok
starting icegrid replica-1... ok
starting icegrid node... ok
starting glacier2... ok
testing login with username/password... ok
testing commands... Traceback (most recent call last):
  File "./run.py", line 140, in <module>
    admin.expect('^active \(.*\)')
  File "/opt/local-10.7-2012-08/var/macports/build/_opt_local-10.7-2012-08_var_macports_sources_rsync.macports.org_release_ports_devel_zeroc-ice35/zeroc-ice35/work/Ice-3.5b/scripts/Expect.py", line 394, in expect
    raise e
Expect.TIMEOUT: timeout exceeded in match
pattern: "^active \\(.*\\)"
buffer: "server state server
active (pid = 9425, enabled)
>>> "

diff -ru Ice-3.4.2.orig/cpp/test/IceGrid/admin/run.py Ice-3.4.2/cpp/test/IceGrid/admin/run.py
--- Ice-3.4.2.orig/cpp/test/IceGrid/admin/run.py	2011-06-15 12:43:59.000000000 -0700
+++ Ice-3.4.2/cpp/test/IceGrid/admin/run.py	2012-12-13 19:39:14.000000000 -0800
@@ -137,7 +137,7 @@
 admin.sendline('server start server')
 admin.expect('>>> ')
 admin.sendline('server state server')
-admin.expect('^active \(.*\)')
+admin.expect('active \(.*\)')
 admin.expect('>>> ')
 admin.sendline('server pid server')
 admin.expect('[0-9]+')
